python/ray/_private/state.py

```diff
@@ -664,6 +664,21 @@ def add_worker(self, worker_id, worker_type, worker_info):
             worker_data.SerializeToString()
         )
 
+    def update_worker_debugger_port(self, worker_id, debugger_port):
```

Comment:
Consider adding an underscore here and for `update_worker_debugger_port` below so it doesn't accidentially end up in a public API / people's autocomplete :)

Reply:
actually looking into this more closely, is this part of the public API. When I look state I find those API's which is differently: https://docs.ray.io/en/latest/ray-observability/reference/api.html

I am working under the expectation that this should only be used in ray internals.

Reply:
I think we expose a few from this `state.py` as public APIS (developers): https://github.com/ray-project/ray/blob/c051a01953230c48cf8ad77edc288b75e7b9ab45/python/ray/__init__.py#L105-L110

Reply:
great, so I will keep as is here since it's not exposed by default